julia/src/model.jl

```diff
@@ -0,0 +1,513 @@
+
+mutable struct StanModelStruct end
+
+"""
+    StanModel(lib, datafile="", seed=204, chain_id=0)
+
+A StanModel instance encapsulates a Stan model instantiated with data.
+
+The constructor a Stan model from the supplied library file path and data file path.
+If seed or chain_id are supplied, these are used to initialize the RNG used by the model.
+
+    StanModel(;stan_file, data="", seed=204, chain_id=0)
+
+Construct a StanModel instance from a `.stan` file, compiling if necessary.
```

Comment:
I don't see where in the code this sentence fits.

Reply:
This overload is actually defined in BridgeStan.jl (line 31) since it requires both this code and the compile code. I tried documenting it there, which worked for `help` inside Julia, but only documented one of them inside Documenter.jl

Reply:
I suppose there is no harm duplicating it there?

Reply:
I see what's going on.  My bad.  Just missed it / didn't put all the pieces together.  Thanks for helping me see the detail.